src/main/java/fact/container/PixelSet.java

```diff
@@ -13,45 +14,56 @@
 import java.util.Set;
 
 /**
- * This is overlay can draw borders around the pixels passed to it via constructor or the add methods.
```

Comment:
What lettered master mind did write this piece of non-sense in the first place ( I fear it was me)